features/logging/deploy_logging_operators_via_console.feature

```diff
@@ -0,0 +1,77 @@
+Feature: Deploy logging operators via console
+
+  # @auther qitang@redhat.com
+  # @case_id OCP-22558
+  @admin
+  @destructive
+  Scenario: Deploy cluster-logging operator via web console.
+    # clear the logging related resources
+    Given logging service is removed successfully
+    Given I switch to cluster admin pseudo user
```

Comment:
switching to admin user not needed. Just do `I run the :create admin command with:` below. And in this way you will not have to use step `I switch to the first user` below.